Fix #37 Moved shelljs out of the global scope

CaptainAchilles opened the pull request.

CaptainAchilles committed:
Moved shelljs out of the global scope

MrCeleryman commented on build/utils.js:
```diff
@@ -69,20 +69,21 @@ exports.styleLoaders = function (options) {
 
 exports.apiSetup = function (options) {
-	var spinner = ora("setting up api...");
+	var spinner = ora("Setting up api...");
```
Good catch!

MrCeleryman commented on build/utils.js:
```diff
@@ -69,20 +69,21 @@ exports.styleLoaders = function (options) {
+	
+	console.log(goOutput.stdout);
+	if (goOutput.stderr) {
+		spinner.stop();		
```
Excellent changes here Nich 🎉

MrCeleryman reviewed (changes requested):
Is shelljs also called in build.js might need to change it there too?

CaptainAchilles committed:
Moved shell out of global scope

Only moves files from static/* if it is a directory and it exists.

CaptainAchilles commented:
Totally didn't see the use of shelljs in the build.js file. I went through and added a check to only copy assets from the static directory if it exists, and cleaned up the global namespace by assigning shelljs to the `shell` var.

MrCeleryman commented:
Great work Nich! So glad you're on board as a collaborator 🥇
